aten/src/ATen/native/TensorTransformations.h

```diff
@@ -9,28 +9,30 @@
 namespace at {
 namespace native {
 
-static inline void flip_check_errors(int64_t total_dims, int64_t flip_dims_size, IntList dims) {
-  // check if number of axis in dim is valid
-  AT_CHECK(flip_dims_size > 0 && flip_dims_size <= total_dims,
-    "flip dims size out of range, got flip dims size=", flip_dims_size);
+static inline void check_dims_errors(IntList dims, int64_t total_dims) {
```

Comment:
What are the semantics of this function? When should I call this function? COmment would be very helpful.

Reply:
it does similar thing as `dim_list_to_bitset`, I will try to remove it in this PR